spec/fixtures/txt/utf8_urls.txt

```diff
@@ -0,0 +1,2 @@
+https://agency.gov/foo § 208.pdf?open
+https://agency.gov/Asesor√≠a.pdf
```

Comment:
I realized that this file also needs a more descriptive name, such as "non_ascii_urls.txt"